website/docs/security-best-practices.md

```diff
@@ -0,0 +1,187 @@
+
+Ethereum's transition to proof-of-stake is made possible by validators who each stake 32 ETH into to the [validator deposit contract](https://docs.prylabs.network/docs/how-prysm-works/validator-deposit-contract/). These validators accept the responsibility to uphold the integrity of the Ethereum network in exchange for staking rewards.
+
+Validators are rewarded for maintaining highly available, trustworthy validator client instances. The security best practices in this guide will help you fulfill this responsibility by helping you minimize risk across a variety of security aspects. Within each aspect, you'll find **recommended**, **advanced**, and **Linux-specific** guidance.
+
+Note that this document is subject to the [Prysmatic Labs Terms of Service](https://github.com/prysmaticlabs/prysm/blob/master/TERMS_OF_SERVICE.md).
+
+
+## Security principles
+The following principles apply generally to staking security:
+
+ - **Keep it simple**. Over-engineered solutions tend to increase risk.
+ - **Stay up to date**. At a minimum, join the [prysm-dev Google Group](https://groups.google.com/g/prysm-dev) to receive important updates related to client security and maintenance. We encourage all stakers to join the Prysm discord and [r/ethstaker](https://www.reddit.com/r/ethstaker). Visit the [Learning Resources](#learning-resources) section at the end of this guide for a short list of resources that we recommend visiting periodically.
+ - **Testnet first**. Harden your configuration using testnet [<a href='#footnote-1'>1</a>] before staking with real ETH on mainnet.
+ - **Simulate risk events**. For each of the aspects within this document, simulate risk events and document your own risk mitigation plans. You can use the [risk mitigation worksheet](#mitigation-worksheet) located at the end of this guide.
+ - **Proactively manage risk** You can't completely eliminate risk, but you can minimize it by following the best practices within this guide.
+ - **If you’re not sure, ask**. The Prysm Discord and [r/ethstaker](https://www.reddit.com/r/ethstaker) subreddit are full of people who genuinely enjoy helping out.
+
+
+
+## Uptime management
+The security of the Ethereum blockchain relies on a **highly available** network of validators. Ethereum's proof-of-stake implementation incentivizes validators to remain online.
+ 
+If your validator goes offline, you can lose some of your staked ETH [<a href='#footnote-2'>2</a>]. As long as you're online most of the time, you'll be profitable. Losses incurred from occasional downtime are negligible [<a href='#footnote-3'>3</a>].
+ 
+While it's possible to optimize your client instance architecture for high-availability and redundancy, we encourage validators to **keep it simple**. Complex validator architectures run the risk of accidentally engaging in malicious behavior. This can result in slashing [<a href='#footnote-4'>4</a>], which is a far steeper price to pay than the occasional downtime penalty.
```

Comment:
instead of `malicious` , maybe `slashable` is better ? Malicious would ascribe intent when there is none other than to run a HA setup.

Reply:
So much better - thanks Nishant. Updated